### ISSUE TITLE
Meta Search Doesn't work since 5.1.3

### ISSUE THREAD

**Issue by Ramox2540** [dbeaver/dbeaver#3863]
The search function in Meta Data doesn't work anymore since 5.1.3
I insert the objectname, select the DB and mark the types i want zu search.

But wenn I press the search button nothing happens. 
![image](https://user-images.githubusercontent.com/41891144/43382350-d37bd1f2-93d7-11e8-8afe-e06d841efdbf.png)

**Comment by serge-rider** [comment]
Yep. It was broken during search engine refactoring and full-text search improvements. Thanks for report.

As a workaround you could try to use GoTo Object feature (<kbd>ctrl+shift+d</kbd>).

**Comment by Ramox2540** [comment]
Thanks for the fast feedback.
Workaround works perfekt for me.
